src/FakeItEasy/Creation/FakeAndDummyManager.cs

```diff
@@ -44,11 +44,14 @@ public FakeAndDummyManager(IDummyValueResolver dummyValueResolver, IFakeObjectCr
 
         public bool TryCreateDummy(Type typeOfDummy, LoopDetectingResolutionContext resolutionContext, out object? result)
         {
-            var creationResult = this.dummyValueResolver.TryResolveDummyValue(typeOfDummy, resolutionContext);
-            if (creationResult.WasSuccessful)
+            if (typeOfDummy != typeof(void))

```

Comment:
How do you feel about instead returning a `default` value in a new strategy in `DummyValueResolver`? It's where we keep all our other special-casing. We could put it above `ResolveFromDummyFactoryStrategy` (perhaps with a comment explaining why it's there).

Functionally it'll be about the same—we'll do something before looking for user Dummy-creators—but I feel like it fits better.

Reply:
Sure, I can do that